tests/integration/helpers/cluster.py

```diff
@@ -868,6 +869,8 @@ def start_zookeeper_nodes(self, zk_nodes):
         cap_add:
             - SYS_PTRACE
             - NET_ADMIN
+            - IPC_LOCK
+            - SYS_NICE
```

Comment:
Since it was working w/o that before i can assume that tested binary has no those capabilities enabled. Not very important here but it's rather better to have all bells and whistles turned in tests.